src/index.ts

```diff
@@ -29,8 +29,9 @@ import {
 } from 'crypto-addr-codec';
 import { sha512_256 } from 'js-sha512';
 import { decode as nanoBase32Decode, encode as nanoBase32Encode } from 'nano-base32';
+import { check } from 'prettier';
```

Comment:
Where is this in use?

Reply:
Seems like I mistakenly added it, fixed.